frontend/src/components/browse/BrowseContent.js

```diff
@@ -116,7 +119,7 @@ export default function BrowseContent({
     TYPES_BY_TAB_VALUE.push("ideas");
   }
   const { locale } = useContext(UserContext);
-  const texts = getTexts({ page: "general", locale: locale });
+  const texts = useMemo(() => getTexts({ page: "general", locale: locale }), [locale]);
```

Comment:
Nit: Good call with `useMemo`! Completely for my own curiosity I'm curious if we were able to measure a perceivable difference in text fetching. I think we should probably be using `useMemo` more widely.

Reply:
Definitely! Well, without it, these texts are computed with each re-render, which sounds a bit creepy. :-)
It's dangerous to try to refactor everything now but maybe it's something we should keep in mind with future changes.